index.ts

```diff
@@ -39,66 +39,74 @@ app.listen(process.env.PORT, () => {
   console.log(`Server listening on port ${port}`);
 });
 
-const executeCmd = async (cmd) => {
+const executeCmd = async (cmd: string) => {
+  let stdout;
+  let stderr;
   try {
-    const { stdout, stderr } = await exec(cmd);
+    const cmdResp = await exec(cmd);
+    stderr  = cmdResp.stderr;
+    stdout = cmdResp.stdout
```

Comment:
please revert the structural change, the extra lines for splitting are unnecessary :)